android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1613,6 +1614,7 @@ private void onBuffering(boolean buffering) {
     @Override
     public void onPositionDiscontinuity(@NonNull Player.PositionInfo oldPosition, @NonNull Player.PositionInfo newPosition, @Player.DiscontinuityReason int reason) {
         if (reason == Player.DISCONTINUITY_REASON_SEEK) {
+            wasSeeking = true;
```

Comment:
It should be false here ? If I am not wrong, onPositionDiscontinuity is called at the end of seek, that's why we send seek end event: "eventEmitter.seek" 
It would make more sens to set `wasSeeking = true;` in `public void seekTo(long positionMs)`.

Not mandatory for the PR, But I also wonder if this seek event shall be reported during initial seeek when you use source.startPosition

Reply:
Correct me if I am wrong - seekTo is our method so controls & external playback management will not be caught in this method 

> Not mandatory for the PR, But I also wonder if this seek event shall be reported during initial seeek when you use 
source.startPosition

yes I think we should not report it in this case

Reply:
Not mandatory for the PR, But I also wonder if this seek event shall be reported during initial seeek when you use
source.startPosition

yes I think we should not report it in this case
--> OK, agreed

Correct me if I am wrong - seekTo is our method so controls & external playback management will not be caught in this method
--> No sure, but yes seekTo is the internal call, but there is also direct call to player.seekTo ...
for exemple if use seek from notifications, will the change work correctly ?

Reply:
Okay - let me re-do this then